src-doc/contenuti/requisiti.tex

```diff
@@ -0,0 +1,160 @@
+
+        \hline
+        RF\_13 & Obbligatorio & L'utente deve poter impostare la modalità dell'area di illuminazione come manuale & Capitolato, UC21\\
+
+        \hline
+        RF\_14 & Obbligatorio & L'utente deve poter impostare la modalità dell'area di illuminazione come automatica & Capitolato, UC22\\
+
+        \hline
+        RF\_15 & Obbligatorio & L'utente deve poter modificare la luminosità dei lampioni del sistema a livello globale & Capitolato, UC23\\
+
+        \hline
+        RF\_16 & Opzionale & Il sistema deve essere in grado di regolare automaticamente l'intensità luminosa dei lampioni in caso rilevi valori fuori soglia di luminosità ambientale & Capitolato\\
+
+        \hline
+        RF\_17 & Opzionale & Il sistema deve essere in grado di rilevare automaticamente guasti nei lampioni & Capitolato \\
+
+        \hline
+        RF\_18 & Obbligatorio & L'utente deve essere in grado di segnalare manualmente guasti nei lampioni & Capitolato, UC6 \\
+
+        \hline
+        RF\_19 & Opzionale & Il gestore deve poter visualizzare l'elenco degli impianti guasti & Capitolato, UC07 \\
+
+        \hline
+        RF\_20 & Obbligatorio & L'utente deve essere in grado di rimuovere guasti dalla lista & Capitolato, UC9 \\
+
+        \hline
+        RF\_20 & Opzionale & Il sistema deve essere in grado di regolare automaticamente l'intensità luminosa dei lampioni in caso rilevi valori fuori soglia di luminosità ambientale & Capitolato\\
+
+        \hline
+        RF\_21 & Obbligatorio & Il sistema deve essere in grado di ricevere informazioni dal sensore in modalità Push & Capitolato \\
+
+        \hline
+        RF\_22 & Opzionale & Il sistema deve essere in grado di ricevere informazioni dal sensore in modalità Pull & Capitolato \\
+
+        \hline
+        RF\_23 & Opzionale & Il sistema visualizzare un messaggio di errore in caso di eccezioni & VIN\_20221129 \\ % non so se metterlo: credo sia corretto ma andrebbe creata l'uc. %
+
+        \hline
+    \end{xltabular}
+\end{center}
+
+
+\section{Requisiti di qualità}
+\begin{center}
+    \begin{xltabular}{\linewidth}{|l|l|X|l|}
+    \hline
+    \rowcolor[gray]{0.9}
+    \textbf{Codice} & \textbf{Classificazione} & \textbf{Descrizione} & \textbf{Fonti} \\
+    \hline
+    RQ\_01 & Obbligatorio & Il progetto deve includere un manuale utente che fornisca spiegazione dettagliate sulle funzionalità dell'applicativo & Capitolato \\
+
+    \hline
+    RQ\_02 & Obbligatorio & Il progetto deve includere un manuale utente che fornisca spiegazione dettagliate su come svolgere manutenzione sull'applicativo & Capitolato \\
+
+    \hline
+    RQ\_03 & Obbligatorio & Il progetto deve essere pubblicato su una repository pubblica & Capitolato \\
+
+    \hline
+    RQ\_04 & Obbligatorio & Il prodotto deve essere sviluppato seguendo le indicazioni definite nelle Norme di Progetto & VIN\_20221129 \\
+
+    \hline
+    RQ\_05 & Obbligatorio & Il codice del progetto deve essere open-source & Capitolato \\
```

Comment:
Da vedere se collima con RQ_03

Reply:
No, dando un'occhiata in giro come linea guida questo è decisamente separato